R/str_indent_wrap.R

```diff
@@ -6,10 +6,10 @@
 #' character element, and use hyphenated wrapping of single words that run
 #' longer than the allotted wrapping width.
 #'
-#' The `stringr::str_wrap()` is extremely highly efficient, but in the context
-#' of table creation there are two select features missing - hyphenation for
-#' long running strings that overflow width, and respect for pre-indentation of
-#' a character element. For example, in an adverse event table, you may have
+#' The function `stringr::str_wrap()` is extremely highly efficient, but in the
```

Comment:
'extremely' and 'highly' feel redundant to me. I say pick one and remove the other.